components/card/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  height: 450px;
+  width: 350px;
+  background-color: white;
+  text-align: center;
+  position: relative;
+}
+
+.card-image {
+  height: auto;
+  width: 100%;
+  border-radius: 0px 0 0 50px;
+  object-fit: cover;
+}
+
+.card-header {
+  margin: 0.2rem auto;
+  padding-top: 0.5rem;
+  font-size: 2rem;
+  color: #474747;
+}
+
+.card-text {
+  display: inline-block;
+  margin: 0.2rem auto;
+  font-size: 1rem;
+  color: #474747;
```

Comment:
You can create variables to maintain consistency. Same for color in line no. 31

Reply:
It's updated now